exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `message`
 
-console.log(message);
+var name = "Aisha"; 
+var nameLength = name.length;
+var message = "My name is " + name + " and the length of name is " +nameLength +" characters long."
```

Comment:
There could be a space before nameLength on this line and perhaps a pronoun before the word name on this section: " and the length of name is "